blocks/test/serializer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { default as serialize } from '../serializer';
+import * as blocks from '../registration';
+
+describe( 'block serializer', () => {
+	// Reset block state before each test.
+	beforeEach( () => {
```

Comment:
In #335 I changed cleanup to `afterEach`. I like `beforeEach` in the general case within a single isolated test suite, but our current setup is not very isolated with singleton blocks, to the effect that one test suite can potentially affect the other. Changing to `afterEach` was an attempt to ensure that each test will not leave any lingering effects to the next.